README.md

```diff
@@ -153,6 +153,36 @@ given buckets** before populating them with test fixture data, please ensure the
 
     tests/fixtures/populate.py --s3-bucket $DSS_S3_BUCKET_TEST_FIXTURES --gs-bucket $DSS_GS_BUCKET_TEST_FIXTURES
 
+#### Set up the whitelist
+
+1. Follow [instructions here](https://github.com/ucsc-cgp/bouncer#setting-up-the-whitelist) to set up the whitelist.
+   The convention for naming the whitelist is `commons/DSS_DEPLOYMENT_STAGE/whitelist`, substituting `DSS_DEPLOYMENT_STAGE`
+   for its value.
+
+2. Then you will need to adjust your environment. Copy this snippet and add it to your `environment.local`
+
+   ```
+   EXPORT_ENV_VARS_TO_LAMBDA_ARRAY+=(
+       EMAIL_WHITELIST_NAME
+       )
+   ```
+
+   Also add the variable `EMAIL_WHITELIST_NAME` to your `environment.local` with the value as
+   the name from the previous step. For example one might add
+
+   ```
+   EMAIL_WHITELIST_NAME=commons/DSS_DEPLOYMENT_STAGE/whitelist
```

Comment:
Please change:
```
EMAIL_WHITELIST_NAME=commons/DSS_DEPLOYMENT_STAGE/whitelist
```
to
```
EMAIL_WHITELIST_NAME=commons/${DSS_DEPLOYMENT_STAGE}/whitelist
```
Both here and where it occurs above.

Reply:
Duh! This is much better!

Reply:
I left the occurrence here https://github.com/DataBiosphere/cgp-data-store/pull/6/files#diff-04c6e90faac2675aa89e2176d2eec7d8R159 because it's what the user will type into AWS, not processed by bash. It's a little more clear this way I think.